easybuild/easyconfigs/p/PnetCDF/PnetCDF-1.9.0-intel-2018a.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'PnetCDF'
+version = '1.9.0'
+
+homepage = 'https://trac.mcs.anl.gov/projects/parallel-netcdf'
+description = """Parallel netCDF: A Parallel I/O Library for NetCDF File Access"""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['http://cucis.ece.northwestern.edu/projects/PnetCDF/Release/']
+sources = ['parallel-netcdf-%(version)s.tar.gz']
+checksums = ['356e1e1fae14bc6c4236ec11435cfea0ff6bde2591531a4a329f9508a01fbe98']
+
+# configopts = '--enable-shared'
```

Comment:
@SimonPinches There's little point in including this (without further explanation)?